src/pages/main/index.tsx

```diff
@@ -36,11 +36,7 @@ const Home = (props: SSRPageProps) => {
 
   return (
     <>
-      <LangSwitcher />
       <Sidebar />
-      <Button variant="contained" onClick={() => logout()}>
-        LogOut
-      </Button>
       <input className={homeStyles.input} />
       <main className={homeStyles.editor}>
```

Comment:
you have main tag in layout, why do you use this tag here one more time

Reply:
I will fix this issue in RG-10